setup.py

```diff
@@ -108,6 +108,10 @@ def file_list(path):
     classifiers=classifiers,
     packages=["hyperdome_server.web", "hyperdome_client",],
     include_package_data=True,
-    scripts=["install/scripts/hyperdome_client", "install/scripts/hyperdome_server"],
+    scripts=[
```

Comment:
`scripts = [`
Looks like the rest of the file is violating pep8 too tho

Reply:
Black is okay with it so I am too